src/main/resources/config.yml

```diff
@@ -11,4 +11,11 @@ contributors:
   - "<Notch><Creator of Minecraft><563.5, 105, 554.5>"
   - "<stxphen><Event Host><559.5, 105, 554.5>"
 
+# Donation Auth
+# Token will be requested periodically using the client ID and secret, which don't expire
+donations:
```

Comment:
Since the token has been moved internally, you can delete this. I appreciate you following up on this though.